weather/contrib/ecowitt/README.md

```diff
@@ -0,0 +1,147 @@
+# localweather Server
+
+![Docker Pulls](https://img.shields.io/docker/pulls/jasonacox/localweather)
+
+This server pulls current weather data from [Ecowitt.net Weather API](https://api.ecowitt.net/api/v3/device/real_time), makes it available via local API calls and stores the data in InfluxDB for graphing.
+
+This service was built to easily add weather data graphs to the [Powerwall-Dashboard](https://github.com/jasonacox/Powerwall-Dashboard) project.
+
+Docker: docker pull [jasonacox/localweather](https://hub.docker.com/r/jasonacox/localweather)
```

Comment:
Assuming others contribute similar services for other local weather stations (or services), I'm thinking we should probably rename the container to match the specific solution it supports.   What about just "ecowitt"?

Reply:
Agree that's a good idea.

Reply:
I'll make those changes tomorrow, and test, and assuming it all works, create another pull request.